src/scss/components/optional/_cookie-warning.scss

```diff
@@ -0,0 +1,73 @@
+@mixin cookieWarning() {
+	.c-cookieWarning {
+	    background-color: $black;
+	    position: fixed;
+	    bottom: 0;
+		width: 100%;
+		$zIndex: high;
```

Comment:
Not sure this will work, as it needs to be a function.  Should be `z-index: zIndex(high);`